salt/grains/zfs.py

```diff
@@ -70,11 +70,12 @@ def _zfs_pool_data():
     return grains
 
 
-def zfs():
+def zfs(grains=None):
```

Comment:
I'm not following how this change is related to Tornado. Could you clarify, please?

Reply:
There was a bug in the beginning of my work that I've fixed this way. I'll check how's it going now and will remove this if it's not needed anymore.